iocore/net/SSLUtils.cc

```diff
@@ -344,6 +361,11 @@ set_context_cert(SSL *ssl)
       ctx = cc->ctx;
   }
 
+  if (cc && cc->hpkp) {
+    netvc->hpkp = cc->hpkp;
```

Comment:
SSLCertContext still owns that. 
Hmm, copying the values of hpkp is only one choice?